spec/notification-manager-spec.js

```diff
@@ -66,4 +66,27 @@ describe('NotificationManager', () => {
       expect(notification.getType()).toBe('success')
     })
   })
+
+  describe('clearing notifications', function () {
```

Comment:
Could you use an arrow function here? That's the style we're using in the rest of the file.